websites/A/animepahe/presence.ts

```diff
@@ -4,60 +4,80 @@
 const presence = new Presence({
     clientId: "629355416714739732" // Contact if you want me to edit the discord assets/keys/whatever
   }),
-  strings = presence.getStrings({
+  waitStrings = async (lang: string) => await presence.getStrings({
     play: "presence.playback.playing",
     pause: "presence.playback.paused",
-    episode: "presence.media.info.episode",
-    browse: "presence.activity.browsing"
-  });
-
-let iframe_response = {
+    browse: "presence.activity.browsing",
```

Comment:
isn't there a general one for it?

Reply:
Its the same string so it really doesn't matter but okay

Reply:
The one you used get removed in v3